### PR TITLE
Add back logging and clloections require bundle

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

**Comment by yanan** [comment]
Fixing tests

File: tools/plugins/com.liferay.ide.core/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -19,7 +19,9 @@ Require-Bundle: org.apache.commons.codec,
  org.eclipse.jdt.launching,
  org.eclipse.jface.text,
  org.eclipse.sapphire.modeling;bundle-version="[9,10)",
- org.json
+ org.json,
+ org.apache.commons.logging,
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.core/META-INF/MANIFEST.MF]: it should be ordered, but I will do it

Comment (yanan) [reply tools/plugins/com.liferay.ide.core/META-INF/MANIFEST.MF]: Got it, thanks.

**Pull request merged**